### DISCUSSION TITLE
Disallow dragging groups in groupbox window

### DISCUSSION THREAD

**Discussion by DasLixou** [qtile/qtile#3792] [Q&A] [answered]
How can i remove the feature to drag groups in the groupbox window?

**Comment by SweepingBishops** [comment, marked as answer]
I don't think there currently is a way. 
Why do you want to disable it? I'm not saying that there shouldn't be a way to do it, I'm just curious.

**Reply by elParaguayo** [reply]
Add `disable_drag=True` in the configuration.
